module/server.go

```diff
@@ -43,6 +43,11 @@ func (s *Server) InstanceNames() []string {
 	return []string{}
 }
 
+// EnsureAuthed is unsupported.
+func (s *Server) EnsureAuthed(ctx context.Context) (context.Context, error) {
+	return nil, errors.New("EnsureAuthed is unsupported")
+}
+
```

Comment:
[q] is this used?

Reply:
It won't be implemented, similar to other functions in this file, they are required to be stubbed because of the interface used. If I take it out:

# go.viam.com/rdk/module
../rdk/module/server.go:24:9: cannot use s (variable of type *Server) as type rpc.Server in return statement:
	*Server does not implement rpc.Server (missing EnsureAuthed method)

Reply:
oops ok!